addons/packages/load-balancer-and-ingress-service/1.8.1/bundle/config/upstream/loadbalancerandingressservice/crds/aviinfrasettings.yaml

```diff
@@ -0,0 +1,107 @@
+#@ load("@ytt:data", "data")
```

Comment:
Please align all the Custom Resource Definitions with https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/release-1.8.1/helm/ako/crds, there may have some updates for adopting new features.